website/src/components/icons/orangeCheckIcon.tsx

```diff
@@ -0,0 +1,12 @@
+export const OrangeCheckIcon = () => (
+  <div className="flex h-5 w-5 items-center justify-center rounded-full bg-orange-500">
```

Comment:
Consider simplify with `size-5`
```suggestion
  <div className="flex size-5 items-center justify-center rounded-full bg-orange-500">
```